ApolloExtra.lua

```diff
@@ -3367,114 +3408,114 @@ OlympusLootData.ApolloUpgrade = {
 			{
                 Name = "ApolloGift01",
 				PlayOnce = true,
-				{ Cue = "/VO/ZagreusHome_0125", Portrait = "Portrait_Zag_Default_01", Speaker = "CharProtag",
+				{ Cue = "/VO/ZagreusHome_3761", Portrait = "Portrait_Zag_Default_01", Speaker = "CharProtag",
 					PreLineAnim = "ZagreusTalkEmpathyStart", PreLineAnimTarget = "Hero",
 					PostLineAnim = "ZagreusTalkEmpathy_Return", PostLineAnimTarget = "Hero",
 					PostLineFunctionName = "BoonInteractPresentation",
-					Text = "Lord Apollo, I am truly blessed simply to have made your acquaintance, so, this is for you." },
-				{ Cue = "/VO/Apollo_0037",
+					Text = "Lord Apollo, you've shown me the way. Please accept this offering." },
+				{ Cue = "/VO/Apollo_0122",
 					PortraitExitWait = 1.0,
 					Emote = "PortraitEmoteAffection",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Lasting relationships are built on reciprocity. So if you lavish me with gifts, why, what am I to do but give you something in return?" },
+					Text = "I'd be lying if I said I didn't see that one coming. That's why I prepared this gift back. Now, we're even." },
 			},
 			ApolloGift02 =
 			{
                 Name = "ApolloGift02",
 				PlayOnce = true,
 				RequiredTextLines = { "ApolloGift01" },
-				{ Cue = "/VO/ZagreusHome_0536", Portrait = "Portrait_Zag_Default_01", Speaker = "CharProtag",
+				{ Cue = "/VO/ZagreusHome_3762", Portrait = "Portrait_Zag_Default_01", Speaker = "CharProtag",
 					PreLineAnim = "ZagreusTalkDenialStart", PreLineAnimTarget = "Hero",
 					PostLineAnim = "ZagreusTalkDenialReturnToIdle", PostLineAnimTarget = "Hero",
 					PostLineFunctionName = "BoonInteractPresentation",
-					Text = "My lord Apollo, if not for our acquaintance, I am certain that my days or nights would be much drearier indeed." },
-				{ Cue = "/VO/Apollo_0034",
+					Text = "Lord Apollo, you're light is very welcome in this darker part of the world. I must thank you for this." },
```

Comment:
"Lord Apollo, your light is very welcome in this darker part of the world. I must thank you for this."